server/src/utils/setSessionIdInCookie.ts

```diff
@@ -0,0 +1,12 @@
+import { ExpressContext } from 'apollo-server-express';
+
+const MAX_AGE_DAYS = 365;
```

Comment:
same than regex validation, should be placed in const file

Reply:
Added in the same enums/Validation.TS